src/reducers/systemIntakesReducer.ts

```diff
@@ -16,18 +18,26 @@ function systemIntakesReducer(
   state = initialState,
   action: Action<any>
 ): SystemIntakesState {
+  const openIntakes: SystemIntakeForm[] = [];
+  const closedIntakes: SystemIntakeForm[] = [];
   switch (action.type) {
     case fetchSystemIntakes.REQUEST:
       return {
         ...state,
         isLoading: true
       };
     case fetchSystemIntakes.SUCCESS:
+      action.payload.forEach((intake: any) => {
```

Comment:
This works, for now, but it is going to fall apart pretty quickly, especially once we add any kind of pagination to the system intakes API. The API is likely faster at doing this (it doesn't sort, it writes a specialized SQL query), and the "open" requests API is also ignoring draft intakes. I really think we need to be making calls to the specific APIs for each of these tables. It's probably ok if we store them in separate reducers, but we shouldn't be doing sorting work on the frontend that the backend can do for us much cheaper.